docs/source/index.mdx

```diff
@@ -19,11 +19,20 @@ As such, Optimum enables developers to efficiently use any of these platforms wi
 
 🤗 Optimum is distributed as a collection of packages - check out the links below for an in-depth look at each one.
 
+
+## Hardware partners
+
+The packages below enable you to get the best of the 🤗 Hugging Face ecosystem on various types of devices.
+
```

Comment:
I like the idea of having an "hardware" section, the only thing that could be unclear would be for packages that also enable hardware agnostic optimization (like INC for example), but this can be specified in the description or even in a specific section, will think of something!

Reply:
Yes, this can be added to the small description in the tiles, or as a note below the tiles such as:

> Some packages provide hardware-agnostic features (e.g. INC interface in Optimum Intel)